tests/asr/test_asr_whisper_on_intel_hpu.sh

```diff
@@ -53,6 +53,18 @@ function validate_microservice() {
         exit 1
     fi
 
+    wget https://github.com/intel/intel-extension-for-transformers/raw/refs/tags/v1.5/intel_extension_for_transformers/neural_chat/ui/customized/talkingbot/src/lib/components/talkbot/assets/mid-age-man.mp3 -O sample.mp3
+    result=$(http_proxy="" curl http://localhost:$ASR_PORT/v1/audio/transcriptions -H "Content-Type: multipart/form-data" -F file="@./sample.mp3" -F model="openai/whisper-small")
+    rm -f sample.mp3
+    if [[ $result == *"welcome to"* ]]; then
+        echo "Result correct."
+    else
+        echo "Result wrong."
+        docker logs whisper-service
```

Comment:
Here should be whisper-gaudi-service/asr-whisper-gaudi-service

Reply:
Currently, dataprep use its own whisper python package, and its own BridgeTowerEmbedding. However, when requesting megaservice, it uses another whisper component microservice and another BridgeTowerEmbedding embedding microservice. Is it possible to simply req/reply the microservices instead of initializing the local equivalent for dataprep?

Reply:
@dmsuehir Sure, sounds good!